fmriprep/cli/run.py

```diff
@@ -217,6 +217,11 @@ def get_parser():
     g_other.add_argument('--stop-on-first-crash', action='store_true', default=False,
                          help='Force stopping on first crash, even if a work directory'
                               ' was specified.')
+    g_other.add_argument('--notrack', action='store_true', default=False,
+                         help='Opt-out of sending tracking information of this run to '
+                              'the FMRIPREP developers. This information helps to '
+                              'improve FMRIPREP and provides an indicator of real '
+                              'world usage crucial for obtaining funding.')
```

Comment:
Not to delay this PR, but maybe by the next release it would be good to add a URL to an entry in the docs where we explicitly state everything that's included in the tracking message, perhaps with the full text of an example message.

I think maximal transparency about things like this will help mollify people who might otherwise opt out.

Reply:
Is there an issue open for this? If not could you open one?

Reply:
Here it is - #1101